errors/ModuleNotFound.md

```diff
@@ -10,12 +10,24 @@ module ShortFailingExample where
 
 ## Cause
 
-Explain why a user might see this error.
+You might see this error because you have tried to import a Module that is not available on your system.
```

Comment:
I think this sentence makes it sound like some modules aren't available on some operating systems; I think a better wording would be "the compiler was unable to find the module you tried to import" or something.